TSQL_Parser/TSQL_Parser/Clauses/TSQLOnClause.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TSQL.Clauses
+{
+	public class TSQLOnClause : TSQLClause

```

Comment:
Is ON a clause, or possibly just an expression? I struggle to define a good rule of thumb if SQL Server official docs don't call it out specifically as a clause.

Reply:
I see ON as being similar to WHERE, which is a clause.
From the Microsoft docs:
FROM JOIN ON: `ON <search_condition>`
SELECT WHERE: `WHERE <search_condition>`
MERGE ON `ON <merge_search_condition>`

This is why I made it a clause but I can change it if you think it should be an expression.